bundles/org.openhab.binding.somfytahoma/src/main/java/org/openhab/binding/somfytahoma/internal/model/SomfyTahomaOauth2Reponse.java

```diff
@@ -0,0 +1,70 @@
+/**
+ * Copyright (c) 2010-2022 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.somfytahoma.internal.model;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+
+/**
+ * The {@link SomfyTahomaOauth2Reponse} holds information about Oauth2 login
+ * response to your CozyTouch account.
+ *
+ * @author Benjamin Lafois - Initial contribution
+ */
+@NonNullByDefault
+public class SomfyTahomaOauth2Reponse {
+    private String scope = "";
+    private String token_type = "";
+    private int expires_in = 0;
+    private String refresh_token = "";
+    private String access_token = "";
```

Comment:
Please do not use `_` in member / variable names.
Thius can be fixed like that:
```
@SerializedName("token_type")
private String tokenType = "";
```

Reply:
done.